File: Estruturas_de_Dados/abstratos/No/No.java
package  Complexo.No;

public class No {
    
    int data;
    No next;

    No(Integer data, No next) {
        this.data = data;
        this.next = next;
    };

    No(Integer data) {

        this.data = data;
        this.next = null;

    };

    public static void main(String[] args) {

        No no = new No(3);
        No no_2 = new No(27);

        no.next = no_2;

        System.out.println(no.data);
        System.out.println(no.next.data);

    };

}

//node 1 -> node 2 -> node 3 -> node 4 -> None
// 4           56       25         86  

File: Estruturas_de_Dados/abstratos/No/no.py
# Essa é a estrutura de dados mais básica que tem

# Como você pode ver, ela tem 2 variaveis: 
# uma variavel de dado, e a outra do próximo node na "lista"

class Node():
    def __init__(self, data):
        
        self.data = data
        self.next = None

no_1 = Node(3)
no_2 = Node(27)

# Aqui, o no_2 ta sendo colocado como o próximo node depois de no_1

no_1.next = no_2

# O código abaixo da erro, no_2 não tem próximo (next)

print(no_1.data)
print(no_1.next.data)
print(no_2.next.data)